config/database.yml

```diff
@@ -0,0 +1,20 @@
+default: &default
```

Comment:
You should create **database.yml.example** file instead

Reply:
@vikabelous done